langchain4j-agentic/src/main/java/dev/langchain4j/agentic/internal/AgentCall.java

```diff
@@ -0,0 +1,15 @@
+package dev.langchain4j.agentic.internal;
+
+import java.util.Arrays;
+
+public record AgentCall(String agentName, Object[] input, Object response) {
```

Comment:
```suggestion
public record AgentCall(String agentName, Object[] input, Object output) {
```
nit

or maybe `result`?

Reply:
Renamed to `output`.